### PR TITLE
add initcontainer to delete lockfiles

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR:

- adds/changes/removes etc

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,12 @@ This application is intended to have a centralized log storage for your applicat
 
 - If you change the name of the helm release from "efk-stack-app" to "logging-stack" you will need to adapt default configuration and change all references of "efk-stack-app-opendistro-es-client-service" to "logging-stack-opendistro-es-client-service" in the [values file](./helm/efk-stack-app/values.yaml)
 
+## Running on NFS
+
+Using NFS for storage is not officially supported by Elasticsearch. Problems can happen when Pods are evicted unexpectedly, lock files will be left and so Elasticsearch in the restarted Pod will complain and stop writing data. As a workaround it is possible to set `elasticsearch.deleteLockfiles.enabled: true` in the value file. In that case an init container will delete all lock files present on the storage for the Elasticsearch data nodes. Please enable this with care! Only enable this when running on NFS and those issues already showd up.
```
Comment (ubergesundheit) [inline README.md:50]: Wasn't this only NFS v3? Is there something to link to?

Comment (ubergesundheit) [reply README.md:50]: The config is `opendistro-es.elasticsearch.deleteLockfiles.enabled: true`, right?

Comment (ubergesundheit) [reply README.md:50]: typo in last sentence

File: README.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ This application is intended to have a centralized log storage for your applicat
 
 - If you change the name of the helm release from "efk-stack-app" to "logging-stack" you will need to adapt default configuration and change all references of "efk-stack-app-opendistro-es-client-service" to "logging-stack-opendistro-es-client-service" in the [values file](./helm/efk-stack-app/values.yaml)
 
+## Running on NFS
+
+Using [NFS for storage is not officially supported](https://discuss.elastic.co/t/why-nfs-is-to-be-avoided-for-data-directories/215240/3) by Elasticsearch. [Problems can happen](https://www.frakkingsweet.com/elasticsearch-nfs-and-locking-issues/) when Pods are evicted unexpectedly, lock files will be left and so Elasticsearch in the restarted Pod will complain and stop writing data. As a workaround it is possible to set `elasticsearch.deleteLockfiles.enabled: true` in the value file. In that case an init container will delete all lock files present on the storage for the Elasticsearch data nodes. Please enable this with care! Only enable this when running on NFS and those issues already showd up.
```
Comment (ubergesundheit) [inline README.md:50]:
```suggestion
Using [NFS for storage is not officially supported](https://discuss.elastic.co/t/why-nfs-is-to-be-avoided-for-data-directories/215240/3) by Elasticsearch. [Problems can happen](https://www.frakkingsweet.com/elasticsearch-nfs-and-locking-issues/) when Pods are evicted unexpectedly, lock files will be left and so Elasticsearch in the restarted Pod will complain and stop writing data. As a workaround it is possible to set `elasticsearch.deleteLockfiles.enabled: true` in the value file. In that case an init container will delete all lock files present on the storage for the Elasticsearch data nodes. Please enable this with care! Only enable this when running on NFS and those issues already showed up.
```

File: README.md (hunk 1 above)
Comment (webwurst) [reply README.md]: Thanks!

**Pull request merged**